### ISSUE TITLE
Runtime does not initialize unless object is manually created.

### ISSUE THREAD

**Issue by trungnt2910** [ikvmnet/ikvm#423]
Suppose that I create a sample `IKVM.NET.Sdk` project using this Java class:

```java
package sample;

public class Sample {
	public static void main(String... args) {
		System.out.println("Hello World");
	}
}
```

And, I reference this from a C# console application:

```csharp
sample.Sample.main(args);
```

I would encounter an exception like this, when trying to access `System.out`:

```
Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at sample.Sample.main(String[] args) in Sample.java:line 5
   at Program.<Main>$(String[] args) in Program.cs:line 1
```

A workaround is to create a dummy object before calling `main()`:

```csharp
// Use a Java Object to ensure the runtime has been initialized.
var obj = new java.lang.Object();

sample.Sample.main(args);
```

**Comment by wasabii** [comment]
So, this is a real problem I've been struggling with for quite awhile: finding the right place to initialize the BCL, without leaving something uncovered, and without causing a bootstrapping issue of some kind.

In real-Java, this is easy. It happens after the VM is initialized, before any user code is run, always, kicked off by java.exe. Since all initialization of the VM goes through this one entry point: java.exe.

But, not the case in .NET, where people can write plain ol' .NET code, and then randomlly call into java classes without going through a centralized loader. Everything I've tried so far has ended up with some sort of bootstrapper issue. Such as a static getting initialized where it shouldn't, and that static needing other things to be initialized, which end up needing that very first static initialized.

**Comment by trungnt2910** [comment]
Maybe making every Java class a child of `java.lang.Object` and then do something in the `java.lang.Object` static constructor may help?

Another potential solution is to have a [module initializer](https://github.com/dotnet/runtime/blob/main/docs/design/specs/Ecma-335-Augments.md#module-initializer) in the IKVM dll, which is run when the IKVM dll is loaded.

**Comment by wasabii** [comment]
This should be resolved in 8.7.4.

**Comment by sergey-tihon** [comment]
Works for me with `v8.7.5`
